src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -4155,15 +4178,28 @@ private BoundExpression BindClassCreationExpression(ObjectCreationExpressionSynt
             }
         }
 
+        internal BoundExpression BindObjectCreationForErrorRecovery(UnboundObjectCreationExpression node, DiagnosticBag diagnostics)
```

Comment:
>BindObjectCreationForErrorRecovery [](start = 33, length = 34)

nit: I suspect the usage in `NullableWalker` could be removed. If that's the case, this method could be made a local function